dss/api/files.py

```diff
@@ -205,11 +205,17 @@ class CopyMode(Enum):
     size = handle.get_size(src_bucket, src_key)
     content_type = handle.get_content_type(src_bucket, src_key)
 
-    # format all the checksums so they're lower-case.
-    for metadata_spec in HCABlobStore.MANDATORY_STAGING_METADATA.values():
-        if metadata_spec['downcase']:
-            keyname = typing.cast(str, metadata_spec['keyname'])
-            metadata[keyname] = metadata[keyname].lower()
+    try:
+        # format all the checksums so they're lower-case.
+        for metadata_spec in HCABlobStore.MANDATORY_STAGING_METADATA.values():
+            if metadata_spec['downcase']:
+                keyname = typing.cast(str, metadata_spec['keyname'])
+                metadata[keyname] = metadata[keyname].lower()
```

Comment:
It may be preferable to narrow the scope of the `try: except` clause to just this line.

Reply:
sure. I'll fix it in another PR.